gcp-blueprint.yaml

```diff
@@ -0,0 +1,262 @@
+tosca_definitions_version: cloudify_dsl_1_3
+
+imports:
+  - https://raw.githubusercontent.com/cloudify-cosmo/cloudify-manager/4.1/resources/rest-service/cloudify/types/types.yaml
+  - http://getcloudify.org.s3.amazonaws.com/spec/gcp-plugin/1.0/plugin.yaml
+  - https://raw.githubusercontent.com/cloudify-incubator/cloudify-utilities-plugin/1.2.5/plugin.yaml
+  - imports/manager-configuration.yaml
+
+inputs:
+
+  gcp_auth_file:
```

Comment:
We need to find a different way of providing authentication details. A file is not secure and not one of our "best practices".